### DISCUSSION TITLE
Getting Modbus data from slave

### DISCUSSION THREAD

**Discussion by frankhvvsp** [WillCodeForCats/solaredge-modbus-multi#500] [General]
Hi, i have a SE5000H-RWS(master) and a SE16K (slave). I get data via Modbus from the master but not for the slave. How can i get data from slave? Thanks

**Comment by WillCodeForCats** [comment]
https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration

**Comment by frankhvvsp** [comment]
So i have to setup the SE16K (slave) with ID=2?

**Reply by WillCodeForCats** [reply]
yes

**Reply by frankhvvsp** [reply]
Now the SE16K has ID=3. But it is not automatically detected. How can i discover it? Thanks

**Comment by frankhvvsp** [comment]
Now the SE16K has ID=3. But it is not automatically detected. How can i discover it? Thanks

**Reply by WillCodeForCats** [reply]
Delete and re-add according to the instructions with the starting ID and the inverter count of 2.

**Comment by knotzchef** [comment]
how do we see it in HA Plugin, that the second inverter is found? I have the same topic and the 2. inverter is conected via MODBus RS485_2 with the 1. inverter......
i have changed the IDs... but i do not see the 2. inverter.

i have 2 3-phase SE10K Inverters

**Reply by WillCodeForCats** [reply]
You have to set the number of inverters in the configuration when you add it to Home Assistant:

Read this:
https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration#integration-configuration
